client/signup/step-wrapper/style.scss

```diff
@@ -55,7 +55,7 @@
 
 	@mixin unstick {
 		position: absolute;
-		top: 1px;
+		top: 8px;
```

Comment:
Most of the spacing increments have been `8px` but this is only `7px`, which will create a minor visual misalignment as seen in the screenshot below. Not sure if it's important, but once I noticed it, I couldn't unsee it 😅

![Screen Shot 2022-07-29 at 6 11 16 PM](https://user-images.githubusercontent.com/797888/181738014-9c26a7ce-90d7-4b32-84ae-25e08343cdbf.png)

Reply:
Should be 9px! 😅